src/main.tsx

```diff
@@ -6,9 +6,10 @@ import { Component, StrictMode } from "react";
 import ReactDom from "react-dom";
 import { BrowserRouter as Router, Route, Routes, Navigate } from "react-router-dom";
 import Header from "./components/header/header";
-import Footer from "./components/footer";
-import Home from "@/pages/home";
+import Footer from "./components/footer/footer";
+import Home from "@/pages/home/home";
 import Products from "@/pages/products/products";
+import ProductsPageWrapper from "@/pages/products/productsPageWrapper";
```

Comment:
Fix all TS errors for this file

Reply:
all errors with error-typing are fixed and error processing checked again